bitnami/opensearch/values.yaml

```diff
@@ -0,0 +1,2342 @@
+    autoGenerated: true
+    ## @param security.tls.verificationMode Verification mode for SSL communications.
+    ## Supported values: full, certificate, none.
+    ## Ref: https://www.open.co/guide/en/opensearch/reference/current/security-settings.html
+    ##
+    verificationMode: "full"
+    ## @param security.tls.master.existingSecret Existing secret containing the certificates for the master nodes
+    ## @param security.tls.data.existingSecret Existing secret containing the certificates for the data nodes
+    ## @param security.tls.ingest.existingSecret Existing secret containing the certificates for the ingest nodes
+    ## @param security.tls.coordinating.existingSecret Existing secret containing the certificates for the coordinating nodes
+    ##
+    master:
+      existingSecret: ""
+    data:
+      existingSecret: ""
+    ingest:
+      existingSecret: ""
+    coordinating:
+      existingSecret: ""
+    ## @param security.tls.keystoreFilename Name of the keystore file
+    ##
+    keystoreFilename: opensearch.keystore.jks
+    ## @param security.tls.truststoreFilename Name of the truststore
+    ##
+    truststoreFilename: opensearch.truststore.jks
+    ## @param security.tls.usePemCerts Use this variable if your secrets contain PEM certificates instead of JKS/PKCS12
+    ## Ignored when using autoGenerated certs.
+    ##
+    usePemCerts: false
+    ## @param security.tls.passwordsSecret Existing secret containing the Keystore and Truststore passwords, or key password if PEM certs are used
+    ##
+    passwordsSecret: ""
+    ## @param security.tls.keystorePassword Password to access the JKS/PKCS12 keystore or PEM key when they are password-protected.
+    ## Ignored if security.tls.passwordsSecret is provided.
+    ##
+    keystorePassword: ""
+    ## @param security.tls.truststorePassword Password to access the JKS/PKCS12 truststore when they are password-protected.
+    ## Ignored if security.tls.passwordsSecret is provided.
+    ##
+    truststorePassword: ""
+    ## @param security.tls.keyPassword Password to access the PEM key when they are password-protected.
+    ## Ignored if security.tls.passwordsSecret is provided.
+    ##
+    keyPassword: ""
+    ## @param security.tls.secretKeystoreKey Name of the secret key containing the Keystore password
+    ##
+    secretKeystoreKey: ""
+    ## @param security.tls.secretTruststoreKey Name of the secret key containing the Truststore password
+    ##
+    secretTruststoreKey: ""
+    ## @param security.tls.secretKey Name of the secret key containing the PEM key password
+    ##
+    secretKey: ""
+    ## @param security.tls.nodesDN A comma separated list of DN for nodes
+    ## e.g. nodesDN: "O=Example CA,C=SE,UID=c-5ca04c9328c8208704310f7c2ed16414"
+    ##
+    ##
+    nodesDN: ""
+    ## @param security.tls.adminDN A comma separated list of DN for admins
+    ##
+    adminDN: ""
+
+## @section Traffic Exposure Parameters
+##
+
+## Opensearch service parameters
+##
+service:
```

Comment:
The NSA checklist expects NetworkPolicies to be available (you don't have to enable by default but at least add the possibility)